orchestrator/handlers/verification_handler.py

```diff
@@ -0,0 +1,18 @@
+from common.logger import get_logger
+from common.utils import handle_exception_with_slack_notification, generate_lambda_response
+from contract_api.config import NETWORK_ID, SLACK_HOOK
+from aws_xray_sdk.core import patch_all
+from orchestrator.services.verification_service import VerificationService
+
+patch_all()
+logger = get_logger(__name__)
+
+Service = VerificationService()
+
+
+@handle_exception_with_slack_notification(logger=logger, SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID)
+def get_fields_handler(event, context):
+    configuration_name = event["pathParameters"]["configurationName"]
+    country_code = event["pathParameters"]["countryCode"]
```

Comment:
add validation for request parameters.

Reply:
https://github.com/singnet/snet-marketplace-service/pull/846/commits/efd99c123ccdb2f29691e165b59f437f16e4a9a7

Reply:
https://github.com/singnet/snet-marketplace-service/pull/846/commits/06e03b7ad0a84d23c48f85b5682ed03b254b75e6